src/experimentExtractor/index.js

```diff
@@ -0,0 +1,128 @@
+const fs = require("fs-extra");
```

Comment:
Thinking about naming — could this dir be `src/webpack`, and the file be `plugin.js`?

Reply:
Now it's `src/experimentExtractor/index.js` 
I will rename it to `src/webpack/index.js` 

I will not rename file, to make it easier to require it.
```js
const {
  default: ExperimentExtractorPlugin,
} = require('next-experiments/dist/webpack');
```

Is it ok?